seqr/views/apis/data_manager_api_tests.py

```diff
@@ -871,7 +945,8 @@ def mock_write(content):
 
                 # test database models are correct
                 self.assertEqual(model_cls.objects.count(), params['initial_model_count'] - deleted_count)
-                sample_guid = self._check_rna_sample_model(individual_id=1, data_source='new_muscle_samples.tsv.gz',
+                sample_guid = self._check_rna_sample_model(individual_id=params.get('exit_sample_individual_id', 1),
+                                                           data_source='new_muscle_samples.tsv.gz',
```

Comment:
The tests for `splice` have a different existing individual id.

Reply:
I really think you should be updating the same individual in the splice test, there is no reason not to

Reply:
The existing sample ID for `outlier` and `TPM` is  `NA19675_D2` and the corresponding individual ID is `NA19675`. The `splice` dataset has an `individual_id` column but no `sample_id` column. We have no way to map the individual ID `NA19675` to the sample ID `NA19675_D2`. So we can't use the same existing sample IDs for the other RNA datasets.

I maybe can add another existing sample model belonging to the same individual as other RNA datasets for the `splice` data.

Reply:
I added a fixture sample with a sample_id `NA19675_1` for the individual NA19675_1 (the same individual of the existing sample models of the Outlier and TPM data). It removes most differences between the `splice` and `outlier/tpm` tests.